app/src/main/java/at/marki/moviedb/MainActivityViewModel.kt

```diff
@@ -21,7 +21,10 @@ import kotlin.time.Duration.Companion.milliseconds
 
 @HiltViewModel
 class MainActivityViewModel @Inject constructor(
+    //REVIEW: why no use case for the login state?
```

Comment:
What I like to do is using UseCase for "OneOff" Logic and Repository is mostly a bundle of similar functions (we also sometimes replaced e.g. 4 UseCases with a repository when it made sense)